Fix range copy.

in SocialHub App

```
java.lang.RuntimeException: java.util.concurrent.ExecutionException: net.socialhub.core.model.error.SocialHubException: java.lang.IllegalArgumentException: 21 > 0
	at 0x000000010472ca38 net.socialhub.core.utils.HandlingUtil.runtime() + 124
	at 0x00000001045489cc SHCAGCommentsRequestGroupActionImpl_$Lambda$4.apply() + 36
	at 0x0000000104f118e8 JavaUtilStreamCollectors_$Lambda$77.accept() + 88
	at 0x0000000104e359f4 java.util.HashMap$EntrySpliterator.forEachRemaining() + 280
	at 0x000000010520f3c4 java.util.stream.AbstractPipeline.copyInto() + 188
	at 0x000000010520f2e8 java.util.stream.AbstractPipeline.wrapAndCopyInto() + 64
	at 0x000000010522fac0 java.util.stream.ReduceOps$ReduceOp.evaluateSequential() + 60
	at 0x0000000105234360 java.util.stream.ReferencePipeline.collect() + 376
	at 0x0000000104548114 net.socialhub.core.action.group.CommentsRequestGroupActionImpl.getComments() + 488
	at 0x00000001045481b0 net.socialhub.core.action.group.CommentsRequestGroupActionImpl.getComments() + 56
	at 0x0000000106a4b6c0 ObjC.catchThrowable:error:() + 60
Caused by: java.util.concurrent.ExecutionException: net.socialhub.core.model.error.SocialHubException: java.lang.IllegalArgumentException: 21 > 0
	at 0x0000000104ed1f44 java.util.concurrent.FutureTask.report() + 108
	at 0x0000000104548948 SHCAGCommentsRequestGroupActionImpl_$Lambda$5.get() + 92
	... 11 more
Caused by: net.socialhub.core.model.error.SocialHubException: java.lang.IllegalArgumentException: 21 > 0
	at 0x0000000104760c70 net.socialhub.core.model.error.SocialHubException.<init>() + 32
	at 0x00000001046a23bc net.socialhub.service.bluesky.action.BlueskyAction.handleException() + 72
	at 0x000000010469f0bc net.socialhub.service.bluesky.action.BlueskyAction.proceed() + 136
	at 0x000000010469f8e0 net.socialhub.service.bluesky.action.BlueskyAction.getHomeTimeLine() + 56
	at 0x0000000104610738 SHCARequestActionImpl_$Lambda$4.apply() + 104
	at 0x0000000104805c18 net.socialhub.core.action.request.CommentsRequestImpl.getComments() + 68
	at 0x00000001045484d8 SHCAGCommentsRequestGroupActionImpl_$Lambda$2.call() + 68
	at 0x0000000104ed2688 java.util.concurrent.FutureTask.run() + 128
	at 0x0000000104ee3134 java.util.concurrent.ThreadPoolExecutor.runTask() + 320
	at 0x0000000104ee2f78 java.util.concurrent.ThreadPoolExecutor.runWorker() + 76
Caused by: java.lang.IllegalArgumentException: 21 > 0
	at 0x0000000104de11b4 java.util.Arrays.copyOfRange() + 176
	at 0x000000010463b298 net.socialhub.service.bluesky.action.BlueskyMapper.getAttributedText() + 1628
	at 0x000000010463a36c net.socialhub.service.bluesky.action.BlueskyMapper.comment() + 1384
	at 0x0000000104639db4 net.socialhub.service.bluesky.action.BlueskyMapper.comment() + 172
	at 0x000000010463df94 SHBsABlueskyMapper_$Lambda$7.apply() + 40
	at 0x0000000105234e1c java.util.stream.ReferencePipeline$$.accept() + 72
	at 0x0000000104ddb784 java.util.ArrayList$ArrayListSpliterator.forEachRemaining() + 256
	at 0x000000010520f3c4 java.util.stream.AbstractPipeline.copyInto() + 188
	at 0x000000010520f2e8 java.util.stream.AbstractPipeline.wrapAndCopyInto() + 64
	at 0x000000010522fac0 java.util.stream.ReduceOps$ReduceOp.evaluateSequential() + 60
	at 0x0000000105234360 java.util.stream.ReferencePipeline.collect() + 376
	at 0x000000010463c3bc net.socialhub.service.bluesky.action.BlueskyMapper.timelineByFeeds() + 384
	at 0x00000001046a4bd0 SHBsABlueskyAction_$Lambda$14.proceed() + 488
```

--- a/src/main/java/net/socialhub/service/bluesky/action/BlueskyMapper.java
+++ b/src/main/java/net/socialhub/service/bluesky/action/BlueskyMapper.java
@@ -28,7 +28,6 @@ import bsky4j.model.bsky.richtext.RichtextFacetLink;
 import bsky4j.model.bsky.richtext.RichtextFacetMention;
 import bsky4j.model.share.RecordUnion;
 import net.socialhub.core.define.MediaType;
-import net.socialhub.core.define.emoji.EmojiCategoryType;
 import net.socialhub.core.model.Channel;
 import net.socialhub.core.model.Comment;
 import net.socialhub.core.model.Identify;
@@ -45,7 +44,6 @@ import net.socialhub.core.model.common.AttributedKind;
 import net.socialhub.core.model.common.AttributedString;
 import net.socialhub.logger.Logger;
 import net.socialhub.service.bluesky.define.BlueskyNotificationType;
-import net.socialhub.service.bluesky.define.BlueskyReactionType;
 import net.socialhub.service.bluesky.model.BlueskyChannel;
 import net.socialhub.service.bluesky.model.BlueskyComment;
 import net.socialhub.service.bluesky.model.BlueskyPaging;
@@ -403,7 +401,7 @@ public class BlueskyMapper {
                         byte[] beforeBytes = copyOfRange(bytes, 0, len);
 
                         readIndex = index.getByteStart();
-                        int afterLen = (bytes.length - len);
+                        int afterLen = Math.max(0, bytes.length - len);
                         bytes = copyOfRange(bytes, len, len + afterLen);
 
                         String str = new String(beforeBytes, StandardCharsets.UTF_8);
@@ -412,6 +410,10 @@ public class BlueskyMapper {
                         element.setExpandedText(str);
                         element.setDisplayText(str);
                         elements.add(element);
+
+                        if (bytes.length == 0) {
+                            break;
+                        }
                     }
 
                     // Facet の部分を切り出して作成
@@ -419,7 +421,7 @@ public class BlueskyMapper {
                     byte[] targetByte = copyOfRange(bytes, 0, len);
 
                     readIndex = index.getByteEnd();
-                    int afterLen = (bytes.length - len);
+                    int afterLen = Math.max(0, bytes.length - len);
                     bytes = copyOfRange(bytes, len, len + afterLen);
 
                     if (union instanceof RichtextFacetMention) {
@@ -448,6 +450,10 @@ public class BlueskyMapper {
                         element.setExpandedText(str);
                         element.setDisplayText(str);
                         elements.add(element);
+                    }
+
+                    if (bytes.length == 0) {
+                        break;
                     }
                 }
             }

